northstar_tests/src/runtime.rs

```diff
@@ -94,7 +94,6 @@ impl Northstar {
             cgroups,
             // This sections matches most common x86 Linux distributions
             devices: config::Devices {
-                unshare_root: PathBuf::from("/"),
```

Comment:
The `unshare_root` is in the example northstar.toml. Think we should add `#[serde(deny_unknown_fields)]` to not allow unknown fields in all struct in `config.rs`.